scripts/self-signed-ca/create-server-cert.sh

```diff
@@ -32,13 +32,13 @@ echo -e "Generating websvr key and csr:\n"
 csr_conf="${this_dir}/conf/server.conf"
 
 if [[ ${dns_name2} == "?" ]]; then
-  san="subjectAltName = DNS:${dns_name1}"
+  export SAN="subjectAltName=DNS:${dns_name1}"
 else
-  san="subjectAltName = DNS:${dns_name1}, DNS:${dns_name2}"
+  export SAN="subjectAltName=DNS:${dns_name1},DNS:${dns_name2}"
 fi
 
 export CN="${dns_name1}"
-openssl req -newkey rsa:2048 -nodes -keyout "${cert_key}" -new -out "${cert_csr}" -config "${csr_conf}" -extensions v3_req -addext "${san}"
```

Comment:
In OpenSSL 3.1+, `-addext` does not play nice with the `v3_req` extension as I think the both contain `subjectAltName` - this causes a duplicate attribute error. This is going to be fixed in OpenSSL 3.4, but that branch isn't going to be in any distro any time soon.

Details: https://github.com/openssl/openssl/issues/11169#issuecomment-2207013616